src/Motor.Extensions.Hosting.RabbitMQ/Options/RabbitMQDeadLetterExchangeOptions.cs

```diff
@@ -0,0 +1,9 @@
+using Motor.Extensions.Hosting.RabbitMQ.Validation;
+
+namespace Motor.Extensions.Hosting.RabbitMQ.Options;
+
+public record RabbitMQDeadLetterExchangeOptions : RabbitMQQueueLimitOptions
+{
+    public string Name { get; set; } = string.Empty;
+    public RabbitMQBindingOptions? Binding { get; set; }
```

Comment:
Any reason why this is nullable? Most usages I see use `Binding!` and therefore enforce that it's not `null`.
Is there a case where it could make sense to have these options but no binding configured?